src/components/_input.scss

```diff
@@ -45,3 +45,9 @@
     box-shadow: 1px solid $c-error;
   }
 }
+
+
+@mixin sui-input-focus() {
```

Comment:
if its something to be used as focus mixin for any element, please remove the word input from the names